transports/quic/src/transport.rs

```diff
@@ -292,17 +289,23 @@ pub(crate) fn socketaddr_to_multiaddr(socket_addr: &SocketAddr) -> Multiaddr {
 mod tests {
     use super::*;
 
+    #[cfg(feature = "noise")]
+    type Crypto = crate::NoiseCrypto;
+    #[cfg(feature = "tls")]
+    type Crypto = crate::TlsCrypto;
```

Comment:
this won't work when both features are enabled

Reply:
Done